target/linux/lantiq/image/xrx330.mk

```diff
@@ -0,0 +1,17 @@
+define Device/dlink_dwr-966
+  $(Device/NAND)
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DWR-966
+  SOC := xrx330
+  DEVICE_PACKAGES := kmod-usb-dwc2 \
+		-ltq-vdsl-vr9-vectoring-fw-installer \
+		-kmod-ltq-vdsl-vr9-mei \
+		-kmod-ltq-vdsl-vr9 \
+		-kmod-ltq-atm-vr9 \
+		-kmod-ltq-ptm-vr9 \
```

Comment:
You are not removing kmod-ltq-deu-vr9 here, although it's also selected in xrx200/target.mk. Is this intended?

Reply:
Yes, DEU is a hardware cryptographic accelerator and is built in xrx300.